### ISSUE TITLE
Wrap for use on Sandstorm

### ISSUE THREAD

**Issue by simonv3** [jdittrich/quickMockup#1]
It would probably be pretty easy to wrap this up as a Sandstorm app and publish it on their store. That provides one-click install for people using Sandstorm, and ease of hosting your own. 

Edit: just going to add this here as a future note to myself: https://github.com/sandstorm-io/sandstorm-rawapi-example

**Comment by jdittrich** [comment]
I am not very knowledgeable in regards to sandstorm. Currently, hosting is as easy as copying the folder to some we accessible location since there is no runtime or server side code needed, so I'm unsure about the benefits of it. If on the other hand, the manual copying of the files is replaced by one click, doing it it is not a bad thing either, as long as it does not complicate anything else.

**Comment by simonv3** [comment]
I don't think it would, but I would have to investigate. I realize that this is as easy as FTPing (or whatever) at the moment, but the easier the better. I'm doing some tweaking of Quick-Survey to get it ready for Sandstorm without impeding its current set up, but after that I can tackle this.

**Comment by jdittrich** [comment]
great; I'm looking forward (and should have a look at QuickSurvey again!)

**Comment by simonv3** [comment]
Eh, it's been a bit stagnant, doing some major refactoring at the moment.

**Comment by TheKnarf** [comment]
Since this app isn't dependent on Sandstorm (and I see no reason why it should ever become so); I recommend closing this issue!

Or do you disagree @simonv3? Are you still working on this?

**Comment by simonv3** [comment]
@TheKnarf I have no opinion on this!
